python/requirements.txt

```diff
@@ -0,0 +1,8 @@
+jupyterlab>=1.2
+matplotlib>=3.1
+pytest>=5
+setuptools>=42
+setuptools_scm>=3.4
+flatbuffers>=1.8.0
+uproot>=3.11.3       
+uproot-methods>=0.7.3  
```

Comment:
These environment.yml and requirements.txt files describe _test_ dependencies. If their file names aren't restricted by anything, I'd like to include "-test" in their names to make that clear. The actual dependencies are all in the setup.py, though.

Reply:
Yep, I am not an expert in these umm.. setup staff. I just installed these packages and see it works. You can do what you say, and I'll double-check later.

Reply:
> Please make the name changes above and then tell me when you're done with the PR.

Ok.